lib/auth/AuthInfo.js

```diff
@@ -44,7 +44,7 @@ export default class AuthInfo {
     }
     // Check whether requester is an IAM user versus an account
     isRequesterAnIAMUser() {
-        return this.IAMdisplayName ? true : false;
+        return !!this.IAMdisplayName;
```

Comment:
Did this actually throw a linter warning? If not, please use the older code as it's more clear and explicit.